### PR TITLE
Generate gdk4-x11

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
Part of #37

### CODE REVIEW

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -24,14 +24,16 @@ jobs:
...
       - run: cargo install rustdoc-stripper
       - run: rustdoc-stripper -g -o ./gdk4/docs.md
       - run: rustdoc-stripper -g -o ./gsk4/docs.md
       - run: rustdoc-stripper -g -o ./gtk4/docs.md
+      - run: rustdoc-stripper -g -o ./gdk4-x11/docs.md
       - uses: actions-rs/cargo@v1
         with:
           command: doc
-          args: -p gtk4 -p gtk4-sys -p gsk4 -p gsk4-sys -p gdk4 -p gdk4-sys --no-deps
+          args: --workspace --no-deps
```
Comment (sdroege) [inline .github/workflows/docs.yml:36]: Is `--workspace` the same as `--all`?

Comment (bilelmoussaoui) [reply .github/workflows/docs.yml:36]: Yes, --all seems to be deprecated

**Comment by sdroege** [comment]
Looks good to me otherwise :)

**Comment by sdroege** [comment]
I.e., merge when you're happy with it and the CI is too

File: .github/workflows/docs.yml (hunk 1 above)
Comment (sdroege) [reply .github/workflows/docs.yml:36]: I should update the GStreamer CI then, thanks!

**Comment by bilelmoussaoui** [comment]
Thanks for the quick review =) 

The CI seems to be happy & it looks fine for an initial thing. I will try to add the gdk4-wayland one as well.

**Pull request merged**